modules/redis/src/main/scala/zio/redis/internal/RedisEnvironment.scala

```diff
@@ -16,13 +16,23 @@
 
 package zio.redis.internal
 
-import zio.redis.CodecSupplier
+import zio.redis.{CodecSupplier, RedisError}
 import zio.schema.Schema
 import zio.schema.codec.BinaryCodec
+import zio.{IO, UIO, ZIO}
 
-private[redis] trait RedisEnvironment {
+private[redis] trait RedisEnvironment[G[+_]] {
   protected def codecSupplier: CodecSupplier
   protected def executor: RedisExecutor
+  protected implicit class RunOps[In, Out](cmd: RedisCommand[In, Out]) {
```

Comment:
I made it final, but I can't make it a value class because it's nested inside `RedisEnvironment`, and it needs to be because it calls the method formerly known as `toG`.